README.md

```diff
@@ -40,7 +40,7 @@ You can install the parcel plugin with [npm](https://www.npmjs.com/get-npm):
 npm install parcel-plugin-elm-symfony-bridge --save-dev
 ```
```

Comment:
Add add the beginning the package version for Webpack and Parcel and a clickable link to each package (https://shields.io/category/version)